examples/building.dae

```diff
@@ -0,0 +1,451 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We have a repo for the examples data. Can you host this there?

Reply:
yes